Vizzuality/landgriffon
# correct blue water footprint calculations

### General description

This PR contains the following:

- Review of the water use and unsustainable water use formulas. This ended up in fixing a matching HS code issues and adding more admin levels for the blue water footprint indicators coefficient data ingestion.
- Review of the current deforestation and carbon formulas. I have updated these formulas in a separate notebook that we can review together and discuss improvements. 

### Testing instructions

For the indicator coefficient data ingestion:

- check that the indicators coefficient data ingestion works successfully
- make sure that the indicator coefficient HScode attributed is ingested as text
- make sure that the indicator coefficient hs code and materials hscode have a successful match (the materialId in the indicators coefficient entity has to be always filled in)




## Checklist before merging
